### DISCUSSION TITLE
Review Iris <-> Xarray docs

### DISCUSSION THREAD

**Discussion by dcherian** [pydata/xarray#7394] [General] [answered]
@pydata/xarray We have a request to comment on https://github.com/SciTools/iris/pull/5025#pullrequestreview-1223902494 which describes the differences between Iris and Xarray in the Iris documentation.

(We should link to it from our docs once its merged)

**Comment by trexfeathers** [comment, marked as answer]
I think this is in a finished state now, so tomorrow I'll start asking around the Iris devs for a merge.
